src/components/LocalFilesValidator.vue

```diff
@@ -23,7 +23,8 @@
   const parallelUpload = (files) => forkJoin(files.map((file) => uploadFile(file, props.workspaceID)));
 
   const uploadFiles = () => {
-    const handleError = () => {
+    const handleError = (error) => {
+      console.error(error);
```

Comment:
Should this `console.error()` call still be here?

Reply:
Good question, I left it in because I thought it might be more informative for the user in the absence of the UI displaying the error message, rather than having to go find the request, but also happy to take it out, what do you think?

Reply:
That makes sense, and good reason to keep in.